charts/daemonset/values.yaml

```diff
@@ -28,6 +31,18 @@ config:
   secrets: {}
   #  username: "password"
 
+  # -- Map of securityContext object applied to pods
+  securityContext: {}
+  
+  # -- Allow pods access to host proccess information
+  hostPID: false
+
+  # -- Allow pods access to host network
+  hostNetwork: false
+
+  # -- DNS Policy for pods
+  dnsPolicy: Default
```

Comment:
Is there a link we could include here that links to the different valid options that could be used?